src/main/resources/db/migration/0250/V259__Modules.sql

```diff
@@ -0,0 +1,39 @@
+ALTER TABLE accelerator.modules ADD COLUMN overview TEXT;
+ALTER TABLE accelerator.modules ADD COLUMN preparation_materials TEXT;
+ALTER TABLE accelerator.modules ADD COLUMN live_session_description TEXT;
+ALTER TABLE accelerator.modules ADD COLUMN workshop_description TEXT;
+ALTER TABLE accelerator.modules ADD COLUMN private_session_description TEXT;
+ALTER TABLE accelerator.modules ADD COLUMN additional_resources TEXT;
+
+ALTER TABLE accelerator.cohort_modules ADD COLUMN phase_id INTEGER REFERENCES accelerator.cohort_phases;
+UPDATE accelerator.cohort_modules SET phase_id = 1 WHERE phase_id IS NULL;
+ALTER TABLE accelerator.cohort_modules ALTER COLUMN phase_id SET NOT NULL;
+
+CREATE TABLE accelerator.event_types (
+    id INTEGER PRIMARY KEY,
+    name TEXT NOT NULL,
+    UNIQUE (name)
+);
+
+CREATE TABLE accelerator.events (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    module_id  BIGINT NOT NULL REFERENCES accelerator.modules ON DELETE CASCADE,
+    event_type_id INTEGER NOT NULL REFERENCES accelerator.event_types,
+    meeting_link TEXT,
+    slides_link TEXT,
+    recording_link TEXT,
+    start_time TIMESTAMP WITH TIME ZONE,
+    end_time TIMESTAMP WITH TIME ZONE,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL,
+
+    UNIQUE(id, module_id, event_type_id)
+);
+
+CREATE TABLE accelerator.event_projects (
```

Comment:
One thing I did not add here is a constraint to make sure a project is not scheduled for multiple events of the same type in the same module.

i.e. it is possible for a project to be in two 1:1 sessions in one module for example. 

We can check for this on the service/controller level I believe. Also give us flexibility if any project does need to be in multiple events (maybe they missed a workshop and joined one later).